lib/driver.js

```diff
@@ -332,20 +333,14 @@ class XCUITestDriver extends BaseDriver {
       await this.stopRemote();
     }
 
-    await this.runReset();
-
     if (this.isSimulator() && this.opts.udid && this.opts.customSSLCert) {
       await uninstallSSLCert(this.opts.customSSLCert, this.opts.udid);
     }
 
     if (this.isSimulator() && !this.opts.noReset && !!this.opts.device) {
-      log.debug('Resetting simulator');
-      if (this.lifecycleData.bootSim) {
```

Comment:
I'd love to change the default behaviour for it, since the main idea here is to shutdown the simulator as rarely as possible.
Do you think think might affect many other things (except of docs)?